flink-connector-hbase-e2e-tests/pom.xml

```diff
@@ -192,14 +192,6 @@ under the License.
 						</goals>
 						<configuration>
 							<artifactItems>
```

Comment:
On line 60-65 we still have an inclusion of `flink-connector-hbase-1.4`

Reply:
See https://github.com/apache/flink-connector-hbase/blob/7c9bda056cdded30daf39ef564e7396e22981c02/flink-connector-hbase-e2e-tests/pom.xml#L60-L65